tests/models/whisper/test_modeling_whisper.py

```diff
@@ -2487,19 +2499,13 @@ def test_default_multilingual_transcription_short_form(self):
 
         transcription = processor.batch_decode(sequences, skip_special_tokens=False)[0]
 
-        assert (
-            transcription
-            == "<|startoftranscript|><|hi|><|transcribe|><|notimestamps|> Mirchi mein ki tene vibinda prajatiya hai<|endoftext|>"
-        )
+        assert transcription == " Mirchi mein ki tene vibinda prajatiya hai"
```

Comment:
(nit)

While we're at it, let's align this with how we assert in the rest of the transformers tests and get rid of `assert`:
```suggestion
       self.assertEqual(transcription, " Mirchi mein ki tene vibinda prajatiya hai")
```

Reply:
applied in commit [c805b19](https://github.com/huggingface/transformers/pull/34111/commits/c805b19f2d310d44fd3d36d5505e87af4179ea41)